### ISSUE TITLE
Make multiple videos on the same story page less likely to fail

### ISSUE THREAD

**Issue by newmuis** [ampproject/amphtml#26634]
Forked from #26630 

There are sometimes valuable reasons to want to have multiple videos per page in an AMP story, which is not well-supported now, and could potentially fail on device/browser combinations where videos are hardware decoded and not enough decoding slots remain.

I wonder: if the videos have literally the same source, do multiple videos still take multiple decoders? In theory we could expose APIs to allow multiple videos in layouts like this (or e.g. the 2-video background blur style) but ONLY allow it if the sources are the same

Or, alternatively, expose APIs that take a single source and produce multiple videos with that source. As a simple example just to get conversation going:

```html
<style amp-custom>
  .vid video:nth-of-type(1) {
    /** Style video 1 */
  }

  .vid video:nth-of-type(2) {
    /** Style video 2 */
  }
</style>

...

<amp-multi-video src="my-video.mp4" count="3" class="vid"></amp-multi-video>
```

This would allow us to ensure that if videos can play at all, then multiple videos can also play, while also allowing the feature set needed here.

I guess the first step would be to verify that having multiple videos with the same src can even be decoded when slots are limited.

**Comment by newmuis** [comment]
@gmajoulet WDYT?

**Comment by mszylkowski** [comment]
@newmuis "multiple videos on the same story page less likely to fail" sounds very similar to the multiple Mediapool issues I closed with #28862. One of the cases where it would fail before, was when you had multiple videos on a late page of the story. This should now be fixed. What should we do with this issue then? (add it to fixit or close it)

**Comment by newmuis** [comment]
I think we can check #26630 and see if it is fixed.  If so, we can close #26630; if not, we add it to fixit.

This is a separate feature request, as a proposed solution to #26630; if it's not necessary (either because #26630 was fixed by #28862, or because we plan to fix it some other way), we can just close this issue.

**Comment by gmajoulet** [comment]
> add it to fixit or close it

Issues should still be able to exist outside of the fixit list. Let's keep the fixit list for what is prioritized. :)) We haven't seen a lot of use cases for multiple videos on a page so far.
